README.md

```diff
@@ -1,27 +1,63 @@
-<img src="man/figures/shiny-react.png" align="right" alt="" width="130" />
+# shiny.react <a href="https://appsilon.github.io/shiny.react/"><img src="man/figures/shiny-react.png" align="right" alt="shiny.react logo" style="height: 140px;"></a>
 
-# shiny.react
+> _Use React in Shiny applications._
 
 <!-- badges: start -->
+[![CRAN
+status](https://www.r-pkg.org/badges/version/shiny.react)](https://cran.r-project.org/package=shiny.react)
+[![cranlogs](https://cranlogs.r-pkg.org/badges/shiny.react)](https://CRAN.R-project.org/package=shiny.react)
+[![total](https://cranlogs.r-pkg.org/badges/grand-total/shiny.react)](https://CRAN.R-project.org/package=shiny.react)
 [![CI](https://github.com/Appsilon/shiny.react/actions/workflows/ci.yaml/badge.svg)](https://github.com/Appsilon/shiny.react/actions/workflows/ci.yaml)
 <!-- badges: end -->
 
-This R package enables using React in Shiny apps and is used e.g. by the shiny.fluent package.
+This R package enables using React in Shiny apps and is used e.g. by the [`shiny.fluent`](https://appsilon.github.io/shiny.fluent/) package.
 It contains R and JS code which is independent from the React library (e.g. Fluent UI) that is being wrapped.
 
-To install the package, run `remotes::install_github("Appsilon/shiny.react")`.
+## Installation
 
-### Development
+Stable version:
+
+```r
+install.packages("shiny.react")
+```
+
+Development version:
+
+```r
+remotes::install_github("Appsilon/shiny.react")
```

Comment:
I'd recommend the stable version to everybody. If you don't know how to install from GitHub, you probably shouldn't be using the development version anyway.

Reply:
I feel that pointing to both versions is kind of a standard, so I prefer to keep the "dev" option, but if you insist, I can remove it.